### PR TITLE
Add file share apps

### PR DESCRIPTION
**Pull request by mzch**
Add file share apps to `Single-click/drag-n-drop upload`

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements (`Demo` and `Clients` are optional, don't add a duplicate `Source code` link if it is the same as the main link):

``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and marked `⊘ Proprietary`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks/specifc dialects.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is "Add aaa to bbb" for adding software aaa to section bbb. Do not call it "Update README.md".

### CODE REVIEW

**Comment by youhost-joseph** [comment]
I've tested an install of shafi (https://github.com/dealfonso/shafi) on Apache and PHP 7.2.
No way to make this work: install.php makes a blank page and manual config works but upload gives error.
I'm afraid this software is unmaintained.
May be the docker works...

File: README.md [hunk 1]
```diff
@@ -744,16 +744,20 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 
 #### Single-click/drag-n-drop upload
 
+- [BlackUpload](https://github.com/FarisCode511/BlackUpload) - An Anonymus File Uploader Script with Multi-File Upload Support. `MIT` `PHP`
 - [BoZoN](https://github.com/broncowdd/BoZoN) - Minimalist Drag and drop file sharing app. `AGPL-3.0` `PHP`
 - [Coquelicot](https://coquelicot.potager.org/) - Coquelicot is a “one-click” file sharing web application with a focus on protecting users’ privacy. ([Source Code](https://coquelicot.potager.org/gitweb/?p=coquelicot.git)) `AGPL-3.0` `Ruby`
 - [elixire](https://elixi.re) - Simple yet advanced screenshot uploading and link shortening service. ([Source Code](https://gitlab.com/elixire/elixire), [Clients](https://gitlab.com/elixire/elixiremanager)) `AGPL-3.0` `Python`
 - [fibridge](https://github.com/anderspitman/fibridge-proxy-rs) - Stream huge files out of your browser without having to upload. `MIT` `Rust`
+- [FireShare](https://auspham.dev/FireShare/) - 🔥A Full-Stack, Pub-Sub, Real-time secure file sharing system. Using NodeJS, ReactJS, SocketIO, MongoDB. Chai, WebdriverIO for testing. `Nodejs` `MIT`
```
Comment (nodiscc) [inline README.md:752]: No source code link, please remove this

File: README.md [hunk 2]
```diff
@@ -744,16 +744,20 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 
 #### Single-click/drag-n-drop upload
 
+- [BlackUpload](https://github.com/FarisCode511/BlackUpload) - An Anonymus File Uploader Script with Multi-File Upload Support. `MIT` `PHP`
```
Comment (nodiscc) [inline README.md:747]: There is no license in the repository, please remove this until a proprer license is added

File: README.md [hunk 3]
```diff
@@ -744,16 +744,20 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
...
 - [Files Sharing](https://github.com/axeloz/filesharing) - Open Source and self-hosted files sharing application based on unique and temporary links. `GPL-3.0` `PHP`
+- [gdprshare](https://github.com/lixmal/gdprshare) - web application to share files with GDPR compliance in mind. `MIT` `Go`
 - [goploader](https://gpldr.in/) - Easy file sharing with server-side encryption, curl/httpie/wget compliant. ([Source Code](https://github.com/Depado/goploader)) `MIT` `Go`
 - [imgpush](https://github.com/hauxir/imgpush) - imgpush is a self-hosted file upload service that can easily be integrated into other webapps. `MIT` `Python`
 - [ipfs.pics](https://github.com/ipfspics/server) - ipfs.pics is a distributed image hosting website. `AGPL-3.0` `PHP`
 - [Jirafeau](https://gitlab.com/mojo42/Jirafeau) - Jirafeau is a web site permitting to upload a file in a simple way and give an unique link to it. ([Demo](http://jirafeau.net/)) `AGPL-3.0` `PHP`
+- [Kleeja](https://kleeja.net/) - ⬇️ File Upload/sharing application, used by thousands of webmasters since 2007. ([Source Code](https://github.com/kleeja-official/kleeja)) `GPL-2.0` `PHP`
```
Comment (nodiscc) [inline README.md:760]: please remove the funky ⬇️ symbol, text descriptions only please

File: README.md [hunk 4]
```diff
@@ -744,16 +744,20 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
...
 - [elixire](https://elixi.re) - Simple yet advanced screenshot uploading and link shortening service. ([Source Code](https://gitlab.com/elixire/elixire), [Clients](https://gitlab.com/elixire/elixiremanager)) `AGPL-3.0` `Python`
 - [fibridge](https://github.com/anderspitman/fibridge-proxy-rs) - Stream huge files out of your browser without having to upload. `MIT` `Rust`
+- [FireShare](https://auspham.dev/FireShare/) - 🔥A Full-Stack, Pub-Sub, Real-time secure file sharing system. Using NodeJS, ReactJS, SocketIO, MongoDB. Chai, WebdriverIO for testing. `Nodejs` `MIT`
 - [FileShelter](https://github.com/epoupon/fileshelter) - FileShelter is a self-hosted software that allows you to easily share files over the Internet. ([Demo](http://fileshelter.demo.poupon.io/)) `GPL-3.0` `C++`
 - [Files Sharing](https://github.com/axeloz/filesharing) - Open Source and self-hosted files sharing application based on unique and temporary links. `GPL-3.0` `PHP`
+- [gdprshare](https://github.com/lixmal/gdprshare) - web application to share files with GDPR compliance in mind. `MIT` `Go`
```
Comment (nodiscc) [inline README.md:755]: reviewed ok

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by nodiscc** [comment]
I reviewed some of the items, please make the required changes. More generally I would advise only adding one or 2 items max. per Pull Request. Reviewing this will take longer

**Comment by mzch** [comment]
OK. I'll make another PR. I've just closed this PR.

**Comment by mzch** [comment]
> I've tested an install of shafi (https://github.com/dealfonso/shafi) on Apache and PHP 7.2.
> No way to make this work: install.php makes a blank page and manual config works but upload gives error.
> I'm afraid this software is unmaintained.
> May be the docker works...

On my instance, ShaFi works completely. Debian 10.7 + Apache 2.4 + PHP 7.3. Perhaps something lacks in your environment.

**Comment by youhost-joseph** [comment]
> On my instance, ShaFi works completely. Debian 10.7 + Apache 2.4 + PHP 7.3. Perhaps your environment lacks something.

Good, I sure lack something in my environment as I tried again but no way. Thank you for pointing this out so people knows it works.

**Pull request closed without merging**